test/DelimitedDataParser.Test/ExporterTest.Reader.cs

```diff
@@ -735,6 +735,36 @@ public void ExportReader_Supports_Null_Column_Name()
             Assert.Equal(@"""Field 1"",""Column2"",""Field 3""", output);
         }
 
+        [Fact]
+        public void ExportReader_Supports_Progress_Report()
```

Comment:
This test seems to have been inserted between tests that are related - test for column name handling. Would it make sense to move this new test to after this group of tests?